.github/workflows/ci-rust.yml

```diff
@@ -101,6 +102,11 @@ jobs:
     - name: Run the tests
       run: cargo tarpaulin --out xml -- --nocapture
 
+    - name: Generate and validate the OpenAPI specification
+      run: |
+        cargo xtask openapi
+        openapi-spec-validator out/openapi/*
```

Comment:
great that it is validated :+1: , but does it also find missing piece?

Reply:
Yes, if it is mentioned and not defined, it is found here (most of the problems were like that). About completely forgotten stuff, we might need to try something different (but I do not know what).